code/src/reducers/todos.js

```diff
@@ -0,0 +1,48 @@
+import { createSlice } from '@reduxjs/toolkit'
+
+export const todos = createSlice({
+    name: 'todos',
+    initialState: {
+        title: 'todo-list',
+        items: [
+        { id: 1,
+          text: 'Watch videos on reducers', 
+          complete: false },
+
+        { id: 2,
+          text: 'Fork weekly assignment', 
+          complete: false },
+
+        { id: 3,
+          text: 'Create a todo app',
+          complete: false },
+      ]
+    },
+    reducers: {
+        //state here is the objects inside the todos
+        removeOne: (state, action) => {
+          const itemId = action.payload;
+          const filteredList = state.items.filter(item => item.id !== itemId)
+          state.items = filteredList;
+        },
+        
+        addOne: (state, action) => {
+          const itemInfo = action.payload;
+          state.items.push(itemInfo)
+        },
+
+        clearAll: (state, action) => {
```

Comment:
You don't need the action param here. Also - suggestion to be consistent with variable names, clear or remove?